client/my-sites/domains/domain-management/transfer/transfer-to-other-site/index.jsx

```diff
@@ -59,9 +61,6 @@ export class TransferToOtherSite extends React.Component {
 			site.capabilities.manage_options &&
 			! ( site.jetpack && ! isAtomic ) && // Simple and Atomic sites. Not Jetpack sites.
 			! get( site, 'options.is_domain_only', false ) &&
-			! (
```

Comment:
This logic change is causing a failure in the unit tests for `TransferToOtherSite.isSiteEligible()`, as it includes a specific test case for the free WPCOM plan.

Reply:
🤦 Crap, missed that - got so used to random tests failing, that I did not notice the unit tests failing for this one. Thanks for pointing it out!